topostats/grains.py

```diff
@@ -295,6 +300,41 @@ def remove_small_objects(self, image: np.array, **kwargs) -> npt.NDArray:
             return small_objects_removed > 0.0
         return image
 
+    def remove_objects_too_small_to_process(
+        self, image: npt.NDArray, minimum_size_px: int, minimum_bbox_size_px: int
```

Comment:
Do we expect this method to be used independently of the processing?

If not then I don't see any advantage have `minimum_size_px` or `minimum_bbox_size_px` as arguments to the method as they are set as attributes to the class when it is initialised and available as `self.minimum_grain_size_px` and `self.minimum_bbox_size_px`.

The comment above on hardcoding suggests its not something that will ever be changed.